default-templates/components/service/languages.php

```diff
@@ -0,0 +1,9 @@
+<?php
+  require_once(__DIR__ . '/../common.php');
+  $languages = $data->service["languages"];
+  echo "<h3>Kielet, joilla palvelu on saatavilla</h3>";
```

Comment:
Could the default template be also localized?

Reply:
Fixed